app/controllers/api/v1/slides_controller.rb

```diff
@@ -0,0 +1,51 @@
+# == Schema Information
+#
+# Table name: slides
+#
+#  id             :integer          not null, primary key
+#  user_id        :integer          not null
+#  name           :string(255)      not null
+#  description    :text(65535)      not null
+#  downloadable   :boolean          default(FALSE), not null
+#  category_id    :integer          not null
+#  created_at     :datetime         not null
+#  modified_at    :datetime
+#  key            :string(255)      default("")
+#  extension      :string(10)       default(""), not null
+#  convert_status :integer          default(0)
+#  total_view     :integer          default(0), not null
+#  page_view      :integer          default(0)
+#  download_count :integer          default(0), not null
+#  embedded_view  :integer          default(0), not null
+#  num_of_pages   :integer          default(0)
+#  comments_count :integer          default(0), not null
+#
+module Api
+  module V1
```

Comment:
__[reek]__
* [UncommunicativeModuleName] has the name 'V1' [:ref](https://github.com/troessner/reek/blob/master/docs/Uncommunicative-Module-Name.md)

<!-- Comment by [SideCI](https://sideci.com) -->